resources/resincli.robot

```diff
@@ -191,21 +206,6 @@ Check if setting environment variables works on "${application_name}"
     Check if "ENV" variable "autohat${random}" with value "RandomValue" exists in application "${application_name}"
     Remove "ENV" variable "autohat${random}" from application "${application_name}"
 
-Check enabling supervisor delta on "${application_name}"
```

Comment:
I assume deleting this breaks the existing test if they try to update to this version of autohat?

So we will rework the tests once this is merged?

Reply:
this was an old test and it isn't used anymore.